src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+section {
+  visibility: hidden;
+}
+
+html {
+  font-family: "Roboto", sans-serif Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+}
+
+.icon {
+  margin: 10px 50px;
+  justify-self: flex-start;
+}
+
+header {
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  align-content: center;
+  justify-self: flex-end;
+  margin: 0;
+}
+
+.margin__nav {
+  position: relative;
+  margin: 0 10px;
+  text-align: center;
+  line-height: 39px;
+  color: black;
+  text-decoration: none;
+  padding: 12px 0;
+
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  color: #00acdc;
```

Comment:
Use CSS variables for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)